styles/styles.css

```
@@ -0,0 +1,696 @@
+  background-size: 100% auto cover;
+  background-position: bottom center;
+  padding-top: 50px;
+  padding-bottom: 80px;
+}
+
+.intro-images {
+  display: block;
+  margin: 0 auto;
+}
+
+/* Advantages */
+.section-advantages {
+  padding-top: 70px;
+  background-color: #ffffff;
+}
+
+.advantages-title {
+  margin: 0 0 25px 0;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 36px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #000000;
+}
+
+.advantages-subtitle {
+  text-align: center;
+  margin: 0 0 62px 0;
+}
+
+.advantages-desc {
+  text-align: center;
+  margin-top: 0;
+  margin-bottom: 64px;
+  font-size: 22px;
+  line-height: 26px;
+}
+
+.advantages-text {
+  text-align: center;
+  margin: 0;
+  font-size: 22px;
+  line-height: 26px;
+}
+
+.advantages-list {
+  margin: 0;
+  padding: 0;
+  display: grid;
+  grid-template-columns: 400px 400px 400px;
+  grid-template-rows: 385px 385px 385px;
+}
+
+.advantages-item {
+  background: rgba(131, 179, 211, 0.12);
+  list-style-type: none;
+
+}
+
+.advantages-wrap {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 102px 64px;
+}
+
+.advantages-wrap-bright {
+  padding: 102px 80px;
+}
+
+.advantages-item-bright {
+  background-color: #82B3D3;
+  display: flex;
+  grid-column-start: 1;
+  grid-column-end: -1;
+  text-align: center;
+
+    color: #ffffff;
+}
+
+.advantages-item-bright .advantages-subtitle {
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 28px;
+  text-transform: uppercase;
+
+}
+
+.advantages-item-active {
+  background: rgba(131, 179, 211, 0.2);
+}
+
+.advantages-item-bright .advantages-text {
+  font-size: 18px;
+  text-align: center;
+  color: #ffffff;
+}
+
+/* Recommend */
+.section-recommend {
+  padding-top: 64px;
+  text-align: center;
+}
+
+.recommend-title {
+  margin: 0 0 20px 0;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 36px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #000000;
+}
+
+.img-icon {
+  margin-bottom: 30px;
+}
+
+.recommend-subtitle {
+  margin-top: 0;
+  margin-bottom: 30px;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 28px;
+  text-transform: uppercase;
+  color: #000000;
+}
+
+.recommend-desc {
+  margin-top: 0;
+  margin-bottom: 64px;
+  font-size: 18px;
+  text-align: center;
+}
+
+.recommend-text {
+  margin: 0;
+  font-size: 18px;
+  text-align: center;
+}
+
+.recommend-list {
+  margin: 0;
+  padding: 0;
+  display: grid;
+  grid-template-columns: 400px 400px 400px;
+}
+
+.recommend-item {
+  list-style-type: none;
+  background: #ffffff;
+}
+
+.recommend-item-wrap {
+  padding: 80px 34px;
+  text-align: center;
+}
+
+.recommend-item-active {
+  background: rgba(131, 179, 211, 0.12);
+}
+
+/* Hotel search */
+.section-hotel-search {
+  background-color: #ffffff;
+  text-align: center;
+}
+
+.hotel-wrap {
+  width: 592px;
+  margin: 0 auto;
+  padding: 96px 0;
+  padding-top: 96px;
+  padding-bottom: 104px;
```

```suggestion
  padding: 96px 0 104px;
```
Да, одно значение может быть удобно переопределить после того как задали со всех сторон. (Я, в большинстве случаев против такого подхода за очень редким исключением).

Но зачем переопределять 96 на 96?

Я за одну строку. 
Либо уж все по отдельности.